internal/quickstart/container.go

```diff
@@ -231,14 +236,14 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 }
 
 func (m ContainerModel) View() string {
-	out := fmt.Sprintf("\nStep %d of %d\n"+m.currentModel.View(), m.currentStep, m.totalSteps)
+	out := fmt.Sprintf("Step %d of %d\n"+m.currentModel.View(), m.currentStep, m.totalSteps)
```

Comment:
Since this is at the very top of the fullscreen now, we should move the newline space from before to after.